docs/sources/reference/commandline/cli.md

```diff
@@ -316,22 +333,28 @@ Currently supported options of `devicemapper`:
 
  *  `dm.loopdatasize`
 
-    Specifies the size to use when creating the loopback file for the "data"
-    device which is used for the thin pool. The default size is 100G. Note that
-    the file is sparse, so it will not initially take up this much space.
+		>**Note**: This option configures devicemapper loopback, which is not for
```

Comment:
Looks like an issue here in the Markdow --- it might have come from me. Can you fix it.

![image](https://cloud.githubusercontent.com/assets/1347209/7820082/e33e34ca-039a-11e5-801c-77c1d091a0fc.png)